### PR TITLE
add note about working with VKCS S3

### PR DESCRIPTION
**Pull request by morozovalekseywot**

### CODE REVIEW

File: templates/index.md.tmpl [hunk 1]
```diff
@@ -47,3 +47,7 @@ provider "vkcs" {
 ```
 
 {{trimattributes .SchemaMarkdown }}
+
+## Working with VKCS S3
```
Comment (ftersin) [inline templates/index.md.tmpl:51]: probably VKCS Cloud Backup

Comment (morozovalekseywot) [reply templates/index.md.tmpl]: done

File: templates/index.md.tmpl [hunk 2]
```diff
@@ -47,3 +47,7 @@ provider "vkcs" {
...
+
+If you want to work with VKCS S3 then follow this [documentation](https://cloud.vk.com/docs/en/tools-for-using-services/terraform/how-to-guides/aws).
```
[the remark this answers is not in the document]
Comment (morozovalekseywot) [reply templates/index.md.tmpl]: done

[the remark this answers is not in the document]
Comment (ftersin) [reply templates/index.md.tmpl]: "a aws"? why the article?

and i'd like to specify that using of aws provider is not a fallback option but a kind of designed solution

i.e. not "so" but "we recommend". or mandatory "use it to". or "the better way to work with .. is to use". etc

[the remark this answers is not in the document]
Comment (morozovalekseywot) [reply templates/index.md.tmpl]: I tried to rewrite it, look again

**Review by ftersin — APPROVED** [review]

**Pull request merged**